java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -300,4 +297,57 @@ void testReplaceNullsFailsOnNullScalar() {
       });
     }
   }
+
+  @Test
+  void testIntegerQuantiles() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    int[] approxExpected = {-1, 1, 1, 2, 7};
+    double[][] exactExpected = {
+        {-1.0,   1.0,   1.0,   2.0,   7.0},
+        {  -1,     1,     1,     2,     7},
+        {  -1,     1,     1,     2,     7},
+        {-1.0,   1.0,   1.0,   2.0,   7.0},
+        {  -1,     1,     1,     2,     7}};
```

Comment:
Could we at least label the different methods?  It is hard to tell what each one is expected to be testing?  I get that we want this kind of data-driven, but I think having clarity in the tests might be preferable.